libs/libfpgabitstream/src/bitstream_manager_utils.cpp

```diff
@@ -130,4 +130,21 @@ size_t rec_find_bitstream_manager_block_sum_of_bits(
   return sum_of_bits;
 }
 
+/********************************************************************
```

Comment:
Please use the split() function provided by openfpga_utils:

https://github.com/lnis-uofu/OpenFPGA/blob/aed082817ed3421ac0cfc45e305ba9191bf457ad/libs/libopenfpgautil/src/openfpga_tokenizer.h#L25

No need to reinvent the wheel

Reply:
split function split the string forward. I need split the string backward. 

Anyway I can live using tokenizer split then reversed the vector.

Done via https://github.com/chungshien/OpenFPGA/commit/2a3d69aded840c54212dd9652670ab54e9ab1222